<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_transcash</class>
 <widget class="QWidget" name="page_transcash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frm_receiptInput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_receiptConfirm">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;정산표를 봉투에 담아
 투입하신 후 확인버튼을 눌러주세요.&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>430</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="pReturnBag">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>도어를 열고, 봉투를 회수 한 후 
  &quot;회수완료&quot; 버튼을 눌러주세요</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ReturnComp">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/base7.png);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>회수완료</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_info_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/IMAGE_DOOROPEN.PNG);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pBarcode">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>우측 하단에 부착된 바코드 리더기에 
새 봉투의 바코드를 인식시켜주세요</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_info_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/IMAGE_BARCODE.PNG);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Barcode">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/base7.png);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>재시도</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pChangeBag">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>봉투를 교체 하시고 도어를 완전히 닫은 후
  &quot;교체완료&quot; 버튼을 눌러주세요.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ChangeComp">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/base7.png);
color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>교체완료</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/IMAGE_DOORCLOSE.PNG);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
